src/ButtonComponent/types.ts

```diff
@@ -9,19 +9,25 @@ export interface ButtonProps extends ButtonConfig {
     globalSettings: GlobalSettings;
 }
 
+type FontWeight = '300' | '400' | '500' | '600' | '700' | '800' | '900';
+export type BorderRadius = 'none' | 'small' | 'medium' | 'large' | 'pill';
```

Comment:
Would it be better to have these as enums?

Reply:
Looked up enums and some articles for them. I still think this is the most readable and useful approach